hoot-core/src/main/cpp/hoot/core/ops/SuperfluousWayRemover.cpp

```diff
@@ -42,13 +42,6 @@ namespace hoot
 
 HOOT_FACTORY_REGISTER(OsmMapOperation, SuperfluousWayRemover)
```

Comment:
Just made this easier to follow and more in line with static methods in other `OsmMapOperation`s